pkg/localkube/dns.go

```diff
@@ -237,11 +237,11 @@ func createEndpoint(client *kubeclient.Client, meta kube.ObjectMeta, dnsIP strin
 				Ports: []kube.EndpointPort{
 					{
 						Name: "dns",
-						Port: dnsPort,
+						Port: int32(dnsPort),
```

Comment:
And to my defense I can say that k8s updated the field to `int32` at HEAD: https://github.com/kubernetes/kubernetes/blob/v1.2.0/pkg/api/types.go#L1641 vs https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L1768

:)